packages/ui-extensions/src/surfaces/point-of-sale/run/input/BaseInput.ts

```diff
@@ -0,0 +1,13 @@
+import type {
+  ConnectivityState,
+  DeviceApiContent,
+  Session,
+} from '../../../point-of-sale';
+import type {CurrencyCode} from '../../../../shared';
+
+export interface BaseInput {
+  connectivity: ConnectivityState;
+  device: DeviceApiContent;
+  locale: string;
+  session: Session & {currency: CurrencyCode};
```

Comment:
I think we should just include currency in the `Session` api. This is a wanted feature for UI rendering extensions as well!